asciidoc-confluence-publisher-converter/src/test/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePageTest.java

```diff
@@ -1089,6 +1090,22 @@ public void renderConfluencePage_asciiDocWithInterDocumentCrossReference_returns
         assertThat(asciidocConfluencePage.content(), is(expectedContent));
     }
 
+    @Test
+    public void renderConfluencePage_asciiDocWithInterDocumentCrossReferenceInSubDirectory_returnsConfluencePageWithLinkToReferencedPageByPageTitleAcknowledgingIncludes() {
```

Comment:
What do you think about this approach?

Reply:
Thank you, that's the test I was looking for :-)